src/cirrus/lib2/process_payload.py

```diff
@@ -22,10 +23,18 @@
 
 
 # clients
+_event_manager = None
 _statedb = None
 _stepfunctions = None
 
 
+def get_event_manager():
+    global _event_manager
+    if _event_manager is None:
+        _event_manager = WorkflowEventManager(logger=logger)
+    return _event_manager
+
+
 def get_statedb():
     global _statedb
     if _statedb is None:
```

Comment:
`WorkflowEventManager` has a `StateDB`, but we are also getting one here; is it worth consolidating and passing the `StateDB` instance in so we only have one?

Reply:
Will do.